### PR TITLE
Fix options not deselecting when nothing selected

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: server/routes/visitors.ts [hunk 1]
```diff
@@ -27,40 +27,40 @@ export default function routes(router: Router, prisonerVisitorsService: Prisoner
...
+      }
+
+      if (selected.length > 3) {
+        throw new Error('Select no more than 3 visitors with a maximum of 2 adults')
+      }
+
+      const adults = req.session.visitorList
+        .filter((visitor: VisitorListItem) => selected.includes(visitor.personId.toString()))
+        .reduce((count: number, visitor: VisitorListItem) => {
+          return visitor.adult ? count + 1 : count
```
Comment (tpmcgowan) [inline server/routes/visitors.ts:51]: Could potentially change to:
`return visitor.adult ?? true ? count + 1 : count`
which I think would stop visitors with no date of birth (so `adult` property is `undefined`) causing issues by effectively counting them as adults - but probably need confirmation on what it *should* do

**Review by tpmcgowan — DISMISSED** [review]
Tested and works OK for me! (Just a comment about potentially how to handle those with no date of birth...)

File: server/routes/visitors.ts (hunk 1 above)
Comment (psoleckimoj) [reply server/routes/visitors.ts]: 👍

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Pull request merged**